centreon/www/class/config-generate/AdditionalConnectorVmWareV6.class.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+/*
+ * Copyright 2005 - 2023 Centreon (https://www.centreon.com/)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * For more information : contact@centreon.com
+ *
+ */
+
+use App\Kernel;
+use Core\AdditionalConnectorConfiguration\Application\Repository\ReadAccRepositoryInterface;
+use Core\AdditionalConnectorConfiguration\Domain\Model\Type;
+use Core\AdditionalConnectorConfiguration\Domain\Model\VmWareV6\{VmWareConfig, VSphereServer};
+
+class AdditionalConnectorVmWareV6 extends AbstractObjectJSON
+{
+    public function __construct(\Pimple\Container $dependencyInjector)
```

Comment:
It's not good practice to inject a Container directly into a constructor. It's better to select the necessary classes

Reply:
The parent needs the Container :/